centos-7-minimal.ks

```diff
@@ -74,47 +72,21 @@ syslinux
 # Setting a global Locale for the server
 echo "LANG=\"C\"" > /etc/locale.conf
 
-# sudo permission
+# Add docker user with 'tcuser' password
```

Comment:
since boot2docker have 'tcuser' as password for the docker user I am choosing same in case it's hard coded in libmachine somewhere.